packages/xod-pm/src/fetch.js

```diff
@@ -69,8 +77,8 @@ export const fetchLibData = R.curry((swaggerUrl, libQuery) =>
     )
 );
 
-// :: URL -> LibName -> Promise Xodball
-export const fetchLibXodball = R.curry((swaggerUrl, libQuery) =>
+// :: URL -> LibName -> Promise Project Error
+export const fetchOneLibrary = R.curry((swaggerUrl, libQuery) =>
```

Comment:
I would rename `fetchOneLibrary/fetchLibrary` pair to something like `fetchLibrary/fetchLibraryAndDependencies`